packages/next/next-server/server/config.ts

```diff
@@ -501,6 +501,10 @@ export default function loadConfig(
 
   // If config file was found
   if (path?.length) {
+    // hook the Node.js require so that webpack requires are
+    // routed to the bundled webpack version
+    require('../../build/webpack/require-hook')
```

Comment:
The issue is most likely here then? Can we at least bisect by reverting this change only, to be sure?

Reply:
Ping @merceyz would be great to see this adjusted to see if that changes the results here. I won't be able to follow up on this personally until next month.

Reply:
Reverted, lets see